tests/integrated/test-kpr_mri/README.md

```diff
@@ -0,0 +1,37 @@
+test-kpr_mri
+===========
+
+ Multirate nonlinear Kvaerno-Prothero-Robinson ODE test problem:
+
+    [f]' = [ G  e ] [(-1+f^2-r)/(2f)] + [      r'(t)/(2f)        ]
+    [g]    [ e -1 ] [(-2+g^2-s)/(2g)]   [ s'(t)/(2*sqrt(2+s(t))) ]
+         = [ fs(t,f,g) ]
+           [ ff(t,f,g) ]
+
+ where r(t) = 0.5 * cos(t),  s(t) = cos(w * t),  0 < t < 5.
+
+ This problem has analytical solution given by
+    f(t) = sqrt(1+r(t)),  g(t) = sqrt(2+s(t)).
+
+ We use the parameters:
+   e = 0.5 (fast/slow coupling strength) [default]
+   G = -100 (stiffness at slow time scale) [default]
+   w = 100  (time-scale separation factor) [default]
```

Comment:
Since these constants are hard-coded in the test problem, there's no point in including "[default]".
```suggestion
   e = 0.5 (fast/slow coupling strength)
   G = -100 (stiffness at slow time scale)
   w = 100  (time-scale separation factor)
```